backend/people/migrations/0017_fill_more_speakers_data.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.26 on 2020-10-22 19:47
+from __future__ import unicode_literals
+from django.db import migrations
+
+def forwards(apps, schema_editor):
+    Speakers = apps.get_model('people', 'Speaker')
+    Speakers.objects.create(
+        name='Mostafa Rezazad',
+        degree='Ph.D.',
+        place='senior researcher at the Institute for Research in Fundamental Sciences',
+        bio='Dr Mostafa Rezazad Received his Ph.D. from the National University of Singapore. Then he joined the CiSDeM group at SUTD university to work on applying AI in the Cybersecurity domain. Now, he is a senior researcher at the Institute for Research in Fundamental Sciences where he is pursuing his research in various directions in the Networking field.'
+    )
```

Comment:
Please fill the `polymorphic_ctype` field of all polymorphic models, such as Speaker, Staff, etc.
Like this:
```Python
from django.contrib.contenttypes.models import ContentType

Speaker = apps.get_model('people', 'Speaker')
ctype = ContentType.objects.get_for_model(Speaker)

Speaker.objects.create(
    name='Mostafa Rezazad',
    degree='Ph.D.',
    place='senior researcher at the Institute for Research in Fundamental Sciences',
    bio='Dr Mostafa Rezazad Received his Ph.D. from the National University of Singapore. Then he joined the CiSDeM group at SUTD university to work on applying AI in the Cybersecurity domain. Now, he is a senior researcher at the Institute for Research in Fundamental Sciences where he is pursuing his research in various directions in the Networking field.',
    polymorphic_ctype=ctype
)
```

Reply:
Done